tests/formattedcode/data/json/tree/expected.json

```diff
@@ -20,6 +20,11 @@
       "size": 91,
       "sha1": "3922760d8492eb8f853c10a627f5a73f9eaec6ff",
       "md5": "fc7f53659b7a9db8b6dff0638641778e",
+<<<<<<< HEAD
```

Comment:
It sounds like you committed unresolved merge conflicts?

Reply:
oh sorry, I must have missed it